types/react-native-auth0/index.d.ts

```diff
@@ -182,39 +180,56 @@ export class Auth {
     multifactorChallenge(params: MultiFactorChallengeParams): Promise<MultiFactorChallengeResponse>;
 }
 
+export interface Auth0Connection {
```

Comment:
Identities wouldn't necessarily just include this structure, You can check the example here - https://auth0.com/docs/manage-users/user-accounts/user-account-linking